xscen/indicators.py

```diff
@@ -293,3 +292,59 @@ def func(ds, *, ind, nout):
 
     func.__name__ = ind.identifier
     return partial(func, ind=ind, nout=nout)
+
+
+def select_inds_for_avail_vars(
+    ds: xr.Dataset,
+    indicators: Union[
+        str,
+        os.PathLike,
+        Sequence[Indicator],
+        Sequence[tuple[str, Indicator]],
+        ModuleType,
+    ],
+) -> ModuleType:
+    """Filter the indicators for which the necessary variables are available.
+
+    Parameters
+    ----------
+    ds : xr.Dataset
+        Dataset to use for the indicators.
+    indicators : Union[str, os.PathLike, Sequence[Indicator], Sequence[Tuple[str, Indicator]]]
+        Path to a YAML file that instructs on how to calculate missing variables.
```

Comment:
missing variables ? I think this probably is a copy-paste from somewhere else, but it is a bit confusing here as in this case the variables are the input of the indicators (defined in the yaml)

Reply:
Good catch. Changed.